playpen/coverage/coverage_hook

```diff
@@ -200,6 +206,75 @@ def _patch_entry_module(install):
                 file.writelines(lines)
 
 
+def _patch_pulp_resource_manager_service(install):
+    """Patch the pulp_resource_manager.service file.
+
+    :param install: boolean that indicates if the patch should be installed or
+        uninstalled, ``True`` or ``False`` respectively.
+    """
+    if install:
+        if _DISTRIBUTION == 'redhat' and _DISTRIBUTION_MAJOR_VERSION == '6':
+            subprocess.check_call([
+                'sed', '-i',
+                '/^VERSION=/ a CELERYD_OPTS="-P solo"',
+                '/etc/rc.d/init.d/pulp_workers',
+            ])
+        else:
+            subprocess.check_call([
+                'sed', '-i',
+                r's/\(--heartbeat-interval=[0-9]\+\).*$/\1 -P solo/',
```

Comment:
This patch needs to be updated if Pulp gets updated since we are editing the source code directly.